### ISSUE TITLE
Option to disable receipts

### ISSUE THREAD

**Issue by travankor** [mirukana/mirage#56]
Riot Web provides this feature. I'm not sure if this is as simple as never sending receipts if enabled.

**Comment by mirukana** [comment]
I can add support for something like this in the config file, to give control over sending receipts:
```json
"sendReadReceipts": {
    "default": true,
    "!some_room_override:example.com": false,
    "@an_account_override:example.com": false,
}
```
I assume this would do what you want? Else, if you can tell me where to find that Riot option and what it does exactly.

**Comment by travankor** [comment]
Yes, that's exactly it. You can see the Riot feature under Settings -> Labs -> Send read receipts for messages.

**Comment by mirukana** [comment]
Related: https://github.com/vector-im/riot-web/issues/2527
Solution to the notification problem used by Riot (needs to be implemented in nio): https://github.com/matrix-org/matrix-doc/pull/2285
